scripts/plugins/githubLinks.js

```diff
@@ -1,24 +1,27 @@
 'use strict'
 
+const sep = require('path').sep
+// add suffix (".html" or sep for windows test) to each part of regex
+// to ignore possible occurrences in titles (e.g. blog posts)
+const isEditable = `(security|index).html|(about|download|docs|foundation|get-involved|knowledge)\\${sep}`
```

Comment:
I still don't see **why** this is needed. These are permalinks, so they will **always** use `/` regardless of the OS.

Reply:
Yes, for Linux and other systems (exclude windows), the original codes work perfectly. However as you see, if for windows (I'm now working on Win, and because it uses '\' instead of '/' as the path sep, so I cannot see the result right or not according to '/' in the gitHubLink.js (because nothing would be found and replaced).

So in order to be compatable with all the platforms, sep is a better choice IMO :)

Reply:
I'm not sure I understand. Are you viewing it with a `file:` URL or something like that?

Reply:
@Trott and @XhmikosR : No. what I mean is: if we do tests on Windows, because the "files" in "scripts\plugins\githubLinks.js" will have the spe = "\", however the original filtering condition is something like "downloads/", this means for windows platform, we cannot get the result (we cannot see "Edit on GitHub" on the locally running test page).

Reply:
I explained it to you... these are links. They will always use `/`. This is not needed.

Reply:
@XhmikosR : I see you mean for the output files, yes, they will always use "/", but on windows platform in gitHhubLinks.js, you'll never get any files to be coped with, because windows is using "\", this means if you using "npm run serve", you will never see the result of "Edit on GitHub" on the page, and you don't know what you did is right or not. 

So considering those who are using windows, I have to use spe instead of hard code here. You can have a try on windows, if you hard coded, then you know why I changed to this :)

Reply:
No, these are links so they will always use `/` regardless of OS. That is why it is working all this time just fine.